public_html/index.html

```diff
@@ -35,6 +35,17 @@
         <h4 class="section-title">Top Stories</h4>
         <ul>
           <li class="highlight__item">
+            <a href="https://hacks.mozilla.org/2017/01/webgl-2-lands-in-firefox/" class="link">
+              <div class="imageWrapper">
+                <img src="assets/img/tile-webgl2.png">
+              </div>
+              <figcaption>
+                <h2 class="highlight__item__title">WebGL2 Lands in Firefox</h2>
+                <p>Tighten up the graphics on level 3 like never before</p>
```

Comment:
expecting to get a rewrite of this text, but [this](http://knowyourmeme.com/memes/tighten-up-the-graphics-on-level-3)